system-tests/lib/crecli/settings.go

```diff
@@ -101,10 +134,15 @@ func PrepareCRECLISettingsFile(workflowOwner, capRegAddr, workflowRegistryAddr c
 				URL:           rpcHTTPURL,
 			},
 		},
+		WorkflowStorage: WorkflowStorage{
+			Gist: Gist{
+				GithubToken: `${CRE_GITHUB_API_TOKEN}`,
```

Comment:
really? now we need to save token in a file? 😞

Reply:
not exactly. It is only reference to env var that stores the token .